packages/cohttp-lwt-jsoo/cohttp-lwt-jsoo.1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+maintainer: "anil@recoil.org"
+authors: [
+  "Anil Madhavapeddy"
+  "Stefano Zacchiroli"
+  "David Sheets"
+  "Thomas Gazagnaire"
+  "David Scott"
+  "Rudi Grinberg"
+  "Andy Ray"
+]
+homepage: "https://github.com/mirage/ocaml-cohttp"
+bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
+license: "ISC"
+tags: ["org:mirage" "org:xapi-project"]
+dev-repo: "https://github.com/mirage/ocaml-cohttp.git"
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "cohttp"
```

Comment:
These all definitely need to be constrained to the same version of Cohttp (1.0) or else testing becomes hard.  At a minimum, `cohttp >= "1.0"` is needed or very old versions of `cohttp` can be selected (this can be seen in the revdeps failures in datakit-ci)

Reply:
Would a 0.99 minimum be a possibility (for those running < 4.06.0).

The ability to upgrade things gradually here could be valuable for people with lots of dependencies.